modules/microvm/common/xdghandlers.nix

```diff
@@ -94,12 +94,19 @@ in
     pdf = lib.mkEnableOption "XDG PDF Handler";
     image = lib.mkEnableOption "XDG Image Handler";
     url = lib.mkEnableOption "XDG Url Handler";
+    urlScript = lib.mkOption {
+      type = lib.types.path;
+      default = "${xdgOpenUrl}/bin/xdgopenurl";
+      description = "Custom script to handle URLs";
+    };
     elementDesktop = lib.mkEnableOption "XDG Element desktop Handler";
   };
 
   config = lib.mkIf config.ghaf.givc.enable {
     environment.systemPackages =
-      (lib.optional cfg.pdf pkgs.zathura) ++ (lib.optional cfg.image pkgs.oculante);
+      (lib.optional cfg.pdf pkgs.zathura)
+      ++ (lib.optional cfg.image pkgs.oculante)
+      ++ (lib.optional cfg.url pkgs.zenity);
```

Comment:
This will add zenity to all VMs that have XDG handlers enabled for URLs. Is it really needed when you use it as `${pkgs.zenity}/bin/zenity` in the script in flatpak-vm?
Also, zenity has quite a lot of dependencies. Maybe you’d be interested in trying yad, which is a lightweight fork with a very similar interface.